logistics/service/v1/data_offer.proto

```diff
@@ -98,6 +99,9 @@ message DataOfferChange {
 
   // Map of properties
   io.spoud.sdm.logistics.mutation.v1.PropertyMap properties = 11;
+
+  // Path
+  google.protobuf.StringValue  path = 13;
```

Comment:
why sometimes `String` other times `StringValue`?

Reply:
For the update/save we sometimes don't want to update all the fields. This is why we need nullable fields.

But for creating or when you retrieve, the non-nullable is preferable.